server/gitlab/api.go

```diff
@@ -111,6 +113,8 @@ func (g *gitlab) NewProjectHook(ctx context.Context, user *UserInfo, token *oaut
 		JobEvents:                &webhookOptions.JobEvents,
 		PipelineEvents:           &webhookOptions.PipelineEvents,
 		WikiPageEvents:           &webhookOptions.WikiPageEvents,
+		DeploymentEvents:         &webhookOptions.DeploymentEvents,
+		ReleasesEvents:           &webhookOptions.ReleaseEvents,
```

Comment:
Should be "ReleaseEvents"?

Reply:
@wiggin77, Similar to https://github.com/mattermost/mattermost-plugin-gitlab/pull/515#discussion_r1709311649